ui/raidboss/data/07-dt/hunts/kozamauka.ts

```diff
@@ -155,7 +156,121 @@ const triggerSet: TriggerSet<Data> = {
       run: (data) => data.nextDoReMisery = [],
       outputStrings: doReMiseryOutputs,
     },
+
     // ****** S-RANK: Ihnuxokiy ****** //
+    {
+      id: 'Hunt Ihnuxokiy Abyssal Smog Initial',
+      type: 'StartsUsing',
+      netRegex: { id: '9B5D', source: 'Ihnuxokiy', capture: false },
+      durationSeconds: 8,
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Hunt Ihnuxokiy Aetherstock Out Collect',
+      type: 'Ability',
+      netRegex: { id: '9B62', source: 'Ihnuxokiy', capture: false },
+      run: (data) => data.nextAetherstock = 'out',
+    },
+    {
+      id: 'Hunt Ihnuxokiy Aetherstock In Collect',
+      type: 'Ability',
+      netRegex: { id: '9B63', source: 'Ihnuxokiy', capture: false },
+      run: (data) => data.nextAetherstock = 'in',
+    },
+    // Ihnuxokiy applies an 8s Forward March/About Face debuff & 12s Left/Right Face debuff.
+    // On expiration, each then converts to a 3s Forced March.
+    // The Abyssal Smog frontal cleave snapshots 1 second after the first Forced March has expired,
+    // just as the Left/Right Face debuff is converting to a Forced March.
+    // The stored Aethersock snapshots as the Left/Right Face Forced March debuff expires.
+    //
+    // Because there is a 1s gap between the end of the Forward/Backward Forced March
+    // and the start of the Left/Right Forced March, players may change their position/facing
+    // during that time.
+    //
+    // The safest way to handle this is probably to call the initial Forward/Backward debuff
+    // with a "(then x)" to indicate that the Left/Right debuff will follow.  As soon as th
+    // Forward/Backward Forced March begins, we can then fire the Left/Right Forced March reminder.
+    //
+    // TODO: Countdowns here would be really useful...
+    {
+      id: 'Hunt Ihnuxokiy Left/Right Face Collect',
+      type: 'GainsEffect',
+      // 873: Left Face, 874: Right Face
+      netRegex: { effectId: ['873', '874'] },
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => data.leftRightFace = matches.effectId === '873' ? 'left' : 'right',
+    },
+    {
+      id: 'Hunt Ihnuxokiy Forced March Forward/Backward',
+      type: 'GainsEffect',
+      // 871: Forward March, 872: About Face
+      netRegex: { effectId: ['871', '872'] },
+      condition: Conditions.targetIsYou(),
+      delaySeconds: 0.2, // let Left/Right Face Collect run first
+      durationSeconds: (_data, matches) => parseInt(matches.duration) - 0.2,
+      infoText: (data, matches, output) => {
+        const foreBack = matches.effectId === '871' ? output.forward!() : output.backward!();
+        const leftRight = output[data.leftRightFace ?? 'unknown']!();
+        return output.combo!({ foreBack: foreBack, leftRight: leftRight });
+      },
+      outputStrings: {
+        combo: {
+          en: 'Forced March: ${foreBack} (then ${leftRight})',
```

Comment:
Why not do `Forced March: ${foreBack} => ${leftRight}` here?

Reply:
In my head at the time, it made sense to do "(then x)" as a way of indicating there was some break between the two forced marches, so that something like turning your character could impact the L/R direction -- as opposed to the two Marches being sequential with no time to intervene. Maybe I'm overthinking it?

Reply:
Given that the delay between them is so short, I'm not sure it makes sense to emphasize the delay more than we would with, say, the `Abyssal Smog Combo`, or other similar sequential combos; i.e., using `=>` is probably sufficient.

Reply:
OK, works for me.